iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/TwtData.java

```diff
@@ -310,8 +345,20 @@ private void calculateThreeConnectedLegsFlow(double u1, double theta1, double u2
         return LinkData.flowBothEnds(admittance.y11, admittance.y12, admittance.y21, admittance.y22, v1, v2);
     }
 
-    private Complex calculateOneConnectedLegFlow(double u, double theta,
-        LinkData.BranchAdmittanceMatrix admittanceMatrixLeg, LinkData.BranchAdmittanceMatrix admittanceMatrixFirstOpenLeg,
+    private Complex calculateTwoConnectedLegsStarBusVoltage(double u1, double theta1, double u2, double theta2,
+        LinkData.BranchAdmittanceMatrix admittanceMatrixLeg1, LinkData.BranchAdmittanceMatrix admittanceMatrixLeg2,
+        LinkData.BranchAdmittanceMatrix admittanceMatrixOpenLeg) {
+
+        Complex v1 = new Complex(u1 * Math.cos(theta1), u1 * Math.sin(theta1));
+        Complex v2 = new Complex(u2 * Math.cos(theta2), u2 * Math.sin(theta2));
```

Comment:
I suggest we use `v1 = ComplexUtils.polar2Complex(u1, theta1)`

Reply:
Done